src/lib/Twig/ContentTaggingExtension.php

```diff
@@ -13,7 +13,7 @@
 use Twig\TwigFunction;
 
 /**
- * Twig content extension for eZ Publish specific usage.
+ * Twig content extension for Ibexa specific usage.
```

Comment:
```suggestion
 * Twig content extension for Ibexa DXP specific usage.
```

Reply:
Not really, applies both to Ibexa Open Source and Ibexa DXP